docs/logs/customizing-the-sdk/README.md

```diff
@@ -26,7 +26,13 @@ See [Program.cs](Program.cs) for an example.
 
 ### IncludeFormattedMessage
 
-TODO
+`ILogger` supports [message templates](https://docs.microsoft.com/dotnet/core/extensions/logging#log-message-template)
```

Comment:
this is irrelevant to `IncludeFormattedMessage`

Reply:
I see that the formatter _can_ include an exception.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/879d3c91a0032eee75061871851e2d4febd6899f/src/OpenTelemetry/Logs/OpenTelemetryLogger.cs#L59

But the default formatter only prints the state:
https://github.com/dotnet/runtime/blob/92b125fc7671b82433f3adbd085b7e6b29b1a73d/src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LoggerExtensions.cs#L425-L428

I haven't been able to find any examples of providing alternative formatters.

Reply:
nevermind, i found it. Caller can invoke `logger.Log` and provide their own formatter. 

I think this is also a testing gap.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/879d3c91a0032eee75061871851e2d4febd6899f/test/OpenTelemetry.Tests/Logs/LogRecordTest.cs#L328-L341